packages/itwinui-css/src/button/base.scss

```diff
@@ -5,6 +5,9 @@
 $disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-iui-disabled="true"]';
 
 @mixin iui-button {
+  --_iui-field-focus-outline-offset: 1px;
+  --_iui-field-focus-outline-width: 2px;
```

Comment:
We don't need this in the base CSS. Let's leave the default unchanged.

It's only the high-visibility and cta buttons that need adjustment.

Reply:
Most of the image changes can also be undone now that the base button is unchanged?

Reply:
Done in https://github.com/iTwin/iTwinUI/pull/2640/commits/7db7d5cf466257c3740765e01747258358c39810.